### ISSUE TITLE
Support jQuery AjaxOption.ContentType property being set with a boolean.

### ISSUE THREAD

**Issue by geoffreymcgill** [bridgedotnet/Bridge#1774]
Forum thread:
http://forums.bridge.net/forum/bridge-net-pro/bugs/2656-open-1774-jquery-ajaxoptions-incorrect-property-type-in-ajax-options

Should be able to set jQuery AjaxOptions.ContentType property with a string or boolean type. 

https://github.com/bridgedotnet/Frameworks/blob/master/jQuery2/Ajax/AjaxOptions.cs#L76-L80

jQuery Docs for `contentType` setting:
http://api.jquery.com/jquery.ajax/#jQuery-ajax-settings

### Expected

Compiles

### Actual

Exception thrown

### Steps To Reproduce

http://deck.net/d96818c4e28a66c069bf0726896ac204

```cs
var options = new AjaxOptions { ContentType = "" };

// Fails
// Some way of setting this client-side property with a boolean is required.
var options = new AjaxOptions { ContentType = false };
```

**Comment by geoffreymcgill** [comment]
How about using `Any<string, bool>`?

The following sample demonstrates:

http://deck.net/96ffe400b8772f7c6c5809d3aa7f5ae7/vert

```cs
public class App
{
    public static void Main()
    {
        var options = new AjaxOptions();
        
        options.ContentType = "test";
        options.ContentType = false;
    }
}

public class AjaxOptions
{
    [FieldProperty]
    public Any<string, bool> ContentType
    {
        get;
        set;
    }
}
```

**Comment by samfromdeath** [comment]
Just tested this on our project (@samuelgrahame), works fine. Thank you

**Comment by geoffreymcgill** [comment]
This issue has been fixed and will be available in the 2.9.2 release of **Bridge.jQuery**.

Your original sample now works:
http://deck.net/d96818c4e28a66c069bf0726896ac204

Hope this helps.
